snippets/getImages.md

```diff
@@ -0,0 +1,17 @@
+### getImages
+
+Fetches all images from within an element and puts them into an array
+
+Use `Element.prototype.getElementsByTagName()` to fetch all `<img>` elements inside `Element`, `Array.prototype.map()` to map every `src` attribute of their respective `<img>` element. `New Set()` creates a `Set` of unique images which is then converted into an `Array` with the `...` operator.
```

Comment:
I'd also write "to fetch all ... elements inside *the provided element*" as `Element` is just a base class.